intg-denonavr/setup_flow.py

```diff
@@ -558,6 +568,28 @@ def __show_all_inputs_cfg(enabled: bool):
     }
 
 
+def __manufacturer_cfg(is_denon: bool):
+    return {
+        "id": "manufacturer",
+        "label": _a("Select manufacturer"),
+        "field": {
+            "dropdown": {
+                "value": "denon" if is_denon else "marantz",
+                "items": [
+                    {
+                        "id": "denon",
+                        "label": _a("Denon"),
+                    },
+                    {
+                        "id": "marantz",
+                        "label": _a("Marantz"),
```

Comment:
Brand names don't need to be translated. I don't think they are called differently in other countries.

Reply:
Dänån, Märäntz, no?

Reply:
hehe